java/client/src/main/java/glide/api/commands/HyperLogLogBaseCommands.java

```diff
@@ -0,0 +1,37 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.commands;
+
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Supports commands and transactions for the "HyperLogLog Commands" group for standalone and
+ * cluster clients.
+ *
+ * @see <a href="https://redis.io/commands/?group=hyperloglog">HyperLogLog Commands</a>
+ */
+public interface HyperLogLogBaseCommands {
+
+    /**
+     * Adds all elements to the HyperLogLog data structure stored at the specified <code>key</code>,
+     * creating a new structure if the <code>key</code> does not exist.
+     *
+     * <p>A command call without <code>elements</code> results in no operation being performed if the
```

Comment:
When no `elements` are provided, and `key` exists and is a HyperLogLog, then no operation is performed.  If `key` does not exist, then the HyperLogLog structure is created.

Reply:
Fixed